restaurant/templates/restaurant/add_food_item.html

```diff
@@ -0,0 +1,292 @@
+                      </li>
+                      <li>
+                        <a href="{% url 'restaurant:category-list' request.restaurant.id %}">Categories</a>
+                      </li>
+                      <li class="active">
+                        <a href="{% url 'restaurant:menu-add' request.restaurant.id %}">Items</a>
+                      </li>
+                      <li>
+                        <a href="#">Modifire Groups</a>
+                      </li>
+                    </ul>
+                  </div>
+                  <div class="fr-pr">
+                    <ul>
+                      <li>
+                        <a href="#"><i class="fa fa-eye" aria-hidden="true"></i>ViewOnline</a>
+                      </li>
+                    </ul>
+                  </div>
+                </div>
+              </div>
+            </div>
+                <!-- right top menu -->
+                <div class="it-ad-hl pd-tb-md">
+                <div class="rb-pd-rl">
+                    <div class="it-ac-hd">
+                    <div class="sc-hd">
+                        <h1>Add Food Item</h1>
+                    </div>
+                    </div>
+                    <!-- heading -->
+                    <div class="ad-fd-fm pd-tb-md">
+                        <form method="POST" enctype="multipart/form-data" action="{% if form.instance.name %}{% url 'restaurant:menu-edit' request.restaurant.id foodmenu.id %}{% else %}{% url 'restaurant:menu-add' request.restaurant.id %}{% endif %}">
+                            {% csrf_token %}
+                            <div class="fm-ls sm-mb">
+                                <label>Food Item Name</label>
+                                {{ form.name }}
+                            </div>
+                            <!-- form list restaurant logo button changed-->
+                            <div class="rest__logo sm-mb">
+                                <div class="fild_inp_logo">
+                                <label>Restaurant Logo</label>
+                                <input type="file" id="imgInp" name="image" />
+                                </div>
+                                <div class="logo_pre">
+                                <img id="blah" src="#" alt=" Restaurant Image" />
+                                </div>
+                            </div>
+                            <!-- form list ingredients change krave  -->
+                            <div class="fm-ls sm-mb">
+                                <label>Description</label>
+                                {{ form.description }}
+                            </div>
+                            <div class="fm-ls sm-mb">
+                                <label>Ingredients</label>
+                                <div class="bootstrap-tagsinput">
+                                    {{ form.ingredients }}
```

Comment:
Does this work as expected in new design? In the new design, we can add multiple ingredients by pressing enter. Do check it.